src/components/types.ts

```diff
@@ -286,3 +288,13 @@ export interface PDFViewerProps {
   /** The url of the PDF file to be displayed. */
   url: string
 }
+
+export interface PerspectiveTableFormat extends DataFormat {
```

Comment:
I think we should use the TableFormat instead of defining yet another DataFormat